tests/data_context/test_data_context.py

```diff
@@ -1783,6 +1783,36 @@ def test_get_batch_multiple_datasources_do_not_scan_all(
     assert len(batch) == 3
 
 
+@mock.patch(
+    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+)
+def test_add_expectation_to_expectation_suite(
+    mock_emit, empty_data_context_stats_enabled
+):
+    context: DataContext = empty_data_context_stats_enabled
+    context.set_data_context(context)
```

Comment:
Here is where we are explicitly setting the `DataContext` registry

Reply:
@Shinnnyshinshin I am starting to lean towards removing this registry, because of the non-uniqueness of the `DataContext` object in a given runtime.  It seemed like a great idea, but now, with the realization of how sometimes multiple `DataContext` objects are instantiated, I worry that it would not work fully.  Thanks.